apps/condo/domains/property/components/panels/Builder/BuildingPanelEdit.tsx

```diff
@@ -273,16 +276,16 @@ export const BuildingPanelEdit: React.FC<IBuildingPanelEditProps> = (props) => {
     }, [mapEdit, refresh])
 
     const menuContent = useMemo(() => ({
-        addSection: <AddSectionForm builder={mapEdit} refresh={refresh}/>,
-        addUnit: <UnitForm builder={mapEdit} refresh={refresh}/>,
-        editSection: <EditSectionForm builder={mapEdit} refresh={refresh}/>,
-        editUnit: <UnitForm builder={mapEdit} refresh={refresh}/>,
+        addSection: <AddSectionForm builder={mapEdit} refresh={refresh} />,
+        addUnit: <UnitForm builder={mapEdit} refresh={refresh} setDuplicatedUnitIds={setDuplicatedUnitIds} />,
```

Comment:
Also you need to pass this state to all of the components as a dependency. But they already contain `MapConstructor` instance